File: app/src/main/java/io/catter2/di/RetrofitCatApiDIModule.java
package io.catter2.di;

import io.catter2.cat_api.TheCatAPI;

public class RetrofitCatApiDIModule extends CachedRetrofitCatApiDIModule {
    @Override
    public TheCatAPI provideTheCatAPI() {
        return super.provideTheCatAPIRetrofit();
    }
}

File: app/src/main/java/io/catter2/di/RetrofitCatApiDIModule.kt
package io.catter2.di

import io.catter2.cat_api.TheCatAPI

class RetrofitCatApiDIModule : CachedRetrofitCatApiDIModule() {
    override fun provideTheCatAPI(): TheCatAPI = super.provideTheCatAPIRetrofit()
}
